tests/blacken_docs_test.py

```diff
@@ -414,14 +414,14 @@ def test_integration_multiple_target_version(tmpdir):
         'def very_very_long_function_name(\n'
         '    very_very_very_very_very_very,\n'
         '    very_very_very_very_very_very,\n'
-        '    *long_long_long_long_long_long\n'
+        '    *long_long_long_long_long_long,\n'
```

Comment:
this no longer demonstrates the behaviour that the option is honored -- the test would "pass" even if no options were being forwarded along

Reply:
That's true, I think I should pass an early-py3 target version instead.

Reply:
Fixed by using py35